src/promise.js

```diff
@@ -236,6 +236,7 @@ Promise.prototype._then = function (
                 receiver = target === this ? undefined : this._boundTo;
             }
         }
+        this._promiseChained(target);
```

Comment:
Right, I've fixed this already in promiseChainLengthLimit branch but forgot to fix in this one, thanks. Will fix now.